autogluon/training/docker/0.3.0/py3/rapids0.19/cu102/Dockerfile.gpu

```diff
@@ -0,0 +1,79 @@
+ARG CUDA_VER=10.2
+ARG LINUX_VER=ubuntu18.04
+ARG PYTHON_VER=3.7
+ARG RAPIDS_VER=0.19
+ARG FROM_IMAGE=rapidsai/rapidsai-core
+
+FROM ${FROM_IMAGE}:${RAPIDS_VER}-cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER}
```

Comment:
What is the OS version used by this image?

Reply:
ubuntu18.04